samples/imagefile_name_category_tags/Military-ControlMeasures-Source-Lines.csv

```diff
@@ -17,7 +17,7 @@ filePath,pointSize,styleItemName,styleItemCategory,styleItemTags,styleItemUnique
 {Symbols_Root}\Appendices\ControlMeasures\25140606.emf,64,Maneuver Lines : Direction of Attack : Enemy Confirmed,Control Measure : Line,Control Measure;Maneuver Lines;Direction of Attack;Enemy Confirmed;gct_arrow;NEW_AT_2525D;MAIN;Line;Maneuver Lines : Direction of Attack : Enemy Confirmed;25140606,25140606,Line,
 {Symbols_Root}\Appendices\ControlMeasures\25140607.emf,64,Maneuver Lines : Direction of Attack : Enemy Templated or Suspected,Control Measure : Line,Control Measure;Maneuver Lines;Direction of Attack;Enemy Templated or Suspected;gct_arrow;NEW_AT_2525D;MAIN;Line;Maneuver Lines : Direction of Attack : Enemy Templated or Suspected;25140607,25140607,Line,
 {Symbols_Root}\Appendices\ControlMeasures\25140700.emf,64,Maneuver Lines : Final Coordination Line,Control Measure : Line,Control Measure;Maneuver Lines;Final Coordination Line;G*GPOLF---****X;MAIN;Line;Maneuver Lines : Final Coordination Line;25140700,25140700,Line,
-{Symbols_Root}\Appendices\ControlMeasures\25140800.emf,64,Maneuver Lines : Infiltration Lane,Control Measure : Line,Control Measure;Maneuver Lines;Infiltration Lane;gct_parallel_lines_mid_line;G*GPOLI---****X;MAIN;Line;Maneuver Lines : Infiltration Lane;25140800,25140800,Line,
+{Symbols_Root}\Appendices\ControlMeasures\25140800.emf,64,Maneuver Lines : Infiltration Lane,Control Measure : Line,Control Measure;Maneuver Lines;Infiltration Lane;gct_parallel_lines_mName_line;G*GPOLI---****X;MAIN;Line;Maneuver Lines : Infiltration Lane;25140800,25140800,Line,
```

Comment:
Good catch.  Fixing now.  Probably the result of a global file-wide search and replace I ran, gone unfortunately wild.